aea/crypto/registries/base.py

```diff
@@ -28,6 +28,7 @@
 
 """A regex to match a Python identifier (i.e. a module/class name)."""
 PY_ID_REGEX = r"[^\d\W]\w*"
+ITEM_ID_REGEX = r"[:/._A-Za-z0-9]+"
```

Comment:
needed a more flexible regex for item ids because I am currently using the public id as contract id (e.g. `fetchai/erc1155:0.5.0`.